### DISCUSSION TITLE
AWS SDK

### DISCUSSION THREAD

**Discussion by eptx** [vlang/v#16092] [Ideas]
Is anyone working on a V AWS SDK? If not, what would be the recommended best approach...transcoding existing cpp AWS SDK to V? See https://github.com/aws/aws-sdk-cpp

**Comment by JalonSolov** [comment]
It would have to be "transcoded" - V has interop with C, but not C++.

**Reply by eptx** [reply]
Go AWS SDK has ~ 1.4M lines of code...lots of documentation in there for sure. I think AWS has tooling to autogenerate a lot of the SDK code. How to put it on their radar?
